openpaas-james/deployment-tests/common/src/main/java/com/linagora/openpaas/deployment/ImapAndSmtpContract.java

```diff
@@ -14,9 +14,9 @@
 import com.google.inject.Injector;
 
 public abstract class ImapAndSmtpContract extends DeploymentValidation {
-    abstract ExternalJamesConfiguration configuration();
+    protected abstract ExternalJamesConfiguration configuration();
 
-    abstract GenericContainer<?> container();
+    protected abstract GenericContainer<?> container();
```

Comment:
somehow I could not extend `DistributedLdapImapAndSmtpTest` with `ImapAndSmtpContract` without making the abstract methods protected... The compilation would just keep failing, while other modules were ok on that... Could not figure out why honestly...